WordPress/src/main/java/org/wordpress/android/support/ZendeskHelper.kt

```diff
@@ -130,29 +121,35 @@ fun showAllTickets(
         zendeskNeedsToBeEnabledError
     }
     getSupportIdentity(context, accountStore.account, selectedSite) { email, name ->
-        configureZendesk(context, email, name, accountStore, siteStore, selectedSite)
-        RequestActivity.startActivity(context, zendeskFeedbackConfiguration(siteStore.sites, origin, extraTags))
+        zendeskInstance.setIdentity(zendeskIdentity(email, name))
+        RequestListActivity.builder()
+                .show(context, zendeskConfig(context, siteStore, origin, selectedSite, extraTags))
     }
 }
 
 // Helpers
 
-private fun configureZendesk(
+private fun zendeskConfig(
```

Comment:
> This also applies to other functions in ZendeskHelper

I think it'd be better if you just comment on them specifically with suggestions on what would be better.

I think we can add `get`, `create` type of prefixes to some of them (although I don't think it'll make it better in most cases), but I don't agree that there is anything else wrong with them. I think you prefer names such as `configureThis` or `doThat` but the functions you commented on so far don't do anything but create and return a value. It's the callers job to take action with them and they are named accordingly as well.

Reply:
Just to make sure there are no misunderstandings, I am both trying to better understand your feedback and explain the parts of the current implementation that might not have been clear to figure out the best approach of the naming.

I am definitely not disregarding your feedback. It hopefully didn't come off that way anyway, but I am kind of paranoid in these type of situations and wanted to be extra sure. Thanks again for the review.

Reply:
No worries :). So the problem I have with non-verb method naming is that it's not immediately clear whether it sets or gets something and you have to look. For example, `zendeskConfig(...)`. Does it create a new Zendesk config? Then `createZendeskConfig()` or `buildZendeskConfig()` would work. Does it just store the supplied values somewhere? (`storeZendeskConfig()`) Does it just retrieve an existing instance (`getZendeskConfig()`)? 

I guess it's not as obvious when you write the code yourself since you know what you meant by it. I'm not trying to be petty about the names. It's just when I see a function named like an object I always have to pause for a moment and think what it does. That's all :).